wagtail/admin/templates/wagtailadmin/shared/header.stories.tsx

```diff
@@ -1,20 +1,16 @@
 import React from 'react';
-import TemplatePattern from '../../../../../client/storybook/TemplatePattern';
+import { Pattern, generateDocs } from 'storybook-django/src/react';
 
 import template from './header.html';
 
+const { docs, argTypes } = generateDocs(template);
+
 export default {
   parameters: {
-    docs: {
-      source: { code: template },
-      // Trial generating documentation from comment within the template. To be replaced by a better pattern.
-      extractComponentDescription: () =>
-        template
-          .match(/{% comment %}\n((.|\n)+){% endcomment %}/m)[1]
-          .replace(/ {4}/g, ''),
-    },
```

Comment:
The moving of this to storybook-django seems to be working OK, but:
1. Why isn't `subtitle` showing up in the Storybook native args table?
2. Only title and icon are currently editable in the args table. Is there a way to make all of the other args editable?

Reply:
> Why isn't subtitle showing up in the Storybook native args table?

Oh! that seems like a bug.

> Only title and icon are currently editable in the args table. Is there a way to make all of the other args editable?

I chose not to because I wasn’t sure how we’d want to go about this. Let’s discuss it again.